cmd/snap-update-ns/entry.go

```diff
@@ -104,3 +104,48 @@ func XSnapdGid(e *mount.Entry) (gid uint64, err error) {
 	}
 	return 0, nil
 }
+
+// XSnapdEntryID returns the identifier of a given mount enrty.
+//
+// Identifiers are kept in the x-snapd.id mount option. The value is a string
+// that identifies a mount entry and is stable across invocations of snapd (it
+// is a hash of some sort).
+func XSnapdEntryID(e *mount.Entry) string {
+	for _, opt := range e.Options {
```

Comment:
This looks quite similar to the helper below. How about moving it to mount.Entry as `OptStr()` or something?

Reply:
Good idea!

Reply:
Done now